package.json

```diff
@@ -33,20 +35,8 @@
       "webpack": false
     }
   ],
-  "repository": {
-    "type": "git",
-    "url": "git+https://github.com/chanzuckerberg/edu-design-system.git"
-  },
-  "keywords": [
-    "CZI",
-    "Learning Platform"
-  ],
   "author": "CZI",
   "license": "MIT",
-  "bugs": {
-    "url": "https://github.com/chanzuckerberg/edu-design-system/issues"
-  },
-  "homepage": "https://github.com/chanzuckerberg/edu-design-system#readme",
```

Comment:
Since this top-level package.json isn't for a published package, we don't need the repo, keywords, bugs, homepage, etc. info.